backend/src/router.js

```diff
@@ -3,12 +3,16 @@ const express = require("express");
 const {
   ItemController,
   UserController,
+  MusicStyleController,
   ArtistController,
+  SearchableController,
 } = require("./controllers");
 
 const router = express.Router();
 
 router.get("/items", ItemController.browse);
+router.get("/musicStyles", MusicStyleController.browse);
+router.get("/artists", ArtistController.browse);
```

Comment:
- not mix /items routes with useful ones
- delete /items routes (only useful as a template)
- use lowercase route names